### PR TITLE
Only residential pay run in create pay run modal

### PR DESCRIPTION
**Pull request by dunnudge**
## Link to CU ticket

[One](https://app.clickup.com/t/1wznx50)
[Two](https://app.clickup.com/t/1wznwxz)

### *Changes introduced*

- Remove direct payment selection
- Remove pay run from date
- Add days since last cycle in ad-hoc tab

### CODE REVIEW

File: components/Pages/Payruns/CreatePayRunModal/AdHocAndReleases.js [hunk 1]
```diff
@@ -3,23 +3,26 @@ import {
...
   RadioGroup,
   VerticalSeparator,
 } from 'components';
-import React from 'react';
+import React, { useMemo } from 'react';
 import { Controller, useForm } from 'react-hook-form';
 import { formValidationSchema } from 'service/formValidationSchema';
+import { addDays, differenceInDays } from 'date-fns';
+import { useLatestPayRunToDate } from '../../../../api';
+
```
Comment (mugnivenko) [inline components/Pages/Payruns/CreatePayRunModal/AdHocAndReleases.js:16]: can be shorthanded as 'api';

remove empty 16 line

**Review by mugnivenko — APPROVED** [review]

File: components/Pages/Payruns/CreatePayRunModal/CreatePayRunModal.js [hunk 2]
```diff
@@ -15,7 +15,8 @@ const CreatePayRunModal = ({ isOpen, onClose, update }) => {
     const data = {
       ...newPayRun,
       paidUpToDate: dateToIsoString(newPayRun.paidUpToDate),
-      paidFromDate: newPayRun.paidFromDate && !newPayRun.isOngoing ? dateToIsoString(newPayRun.paidFromDate) : null,
+      // paidFromDate: newPayRun.paidFromDate && !newPayRun.isOngoing ? dateToIsoString(newPayRun.paidFromDate) : null,
```
Comment (mugnivenko) [inline components/Pages/Payruns/CreatePayRunModal/CreatePayRunModal.js:18]: remove commentary

Comment (dunnudge) [reply components/Pages/Payruns/CreatePayRunModal/CreatePayRunModal.js]: Done

File: components/Pages/Payruns/CreatePayRunModal/AdHocAndReleases.js (hunk 1 above)
Comment (dunnudge) [reply components/Pages/Payruns/CreatePayRunModal/AdHocAndReleases.js]: Updated

**Pull request merged**